src/azul/security/auth.py

```diff
@@ -0,0 +1,80 @@
+# See https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/Security/Authentication%20and%20Authorization/Setting%20up%20DCP%20Auth
+from logging import getLogger
+from typing import Dict, Any
+from urllib.parse import urlencode
+
+from azul import config
+from azul.security.verifier import (verify,
+                                    InvalidTokenError,
+                                    NonDecodableTokenError)
+
+logger = getLogger(__name__)
+fusillade_callback_url = config.service_endpoint() + '/auth/callback'
```

Comment:
This seems to be factored into a constant for patching during testing. I don't like it because it essentially caches  the return value of `config.service_endpoint` when the module is imported. Changes to the underlying environment variables from which `config.service_endpoint` is derived will not be reflected. This is why everything in Config is a property and Config is stateless. This type of caching breaks that.